src/utils/passPhraseToRpc.ts

```diff
@@ -0,0 +1,22 @@
+import { Futurenet, Mainnet, Testnet } from 'src/constants/networks';
+
+const passPhraseToNetworkURL = (passPhrase: string) => {
+  if (passPhrase === Mainnet.networkPassphrase) {
+    return {
+      sorobanRPC: Mainnet.sorobanRpcUrl,
+      horizonRPC: Mainnet.networkUrl,
```

Comment:
add contract field to each of them and change the file and function name to passPhraseToConfig